transforms/index.js

```diff
@@ -52,10 +55,38 @@ exports.afterParse = function(parser) {
     
       // add the new function to the AST as an exported function
       entrySrc.statements.push(callWrapperStmt);
+    }
+
+  })
+
+  // this will need to be changed to support multiple capabilities
+  writeZomeJSON(zomeFuncs, '../capabilities/main/autogen.zome.json')
+}
 
-      // TODO: add some data to the manifest.json
 
+function writeZomeJSON(zomeFuncs, path) {
+  let zome;
+  if (fs.existsSync(path)) { // update an existing zome.json
+    let rawJSON = fs.readFileSync(path);
+    zome = JSON.parse(rawJSON);
+  } else { // have to create a new one from scratch
+
+    zome = {
+      description: "",
+      capability: {
+        membrane: "agent"
+      },
+      functions: []
     }
+  }
 
-  })
+  zome.functions = zomeFuncs.map(func => {
+    return {
+      name: func.name,
+      inputs: func.params,
+      outputs: []
+    }
+  });
+
+  fs.writeFileSync(path, JSON.stringify(zome, null, 2));
```

Comment:
Oh! We should create the directory/path if it doesn't exist. Unless this command already does this but I doubt it. 

Then they don't need to create `capabilities/main` by hand